frontend/src/components/common/Langswitcher.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import { locale, locales } from 'svelte-i18n';
+  import { isLoading } from 'svelte-i18n';
+
+  function handleChange(event: Event) {
+    const selectedLocale = (event.target as HTMLSelectElement).value;
+    localStorage.setItem('locale', selectedLocale);
+  }
+
+  if ($locale != null && $locale != undefined && $locale.startsWith('ru')) {
+    $locale = 'ru';
+  } else {
+    $locale = 'en';
+  }
```

Comment:
I feel like it should be like 

```suggestion
  if (!!$locale) $locale = 'en';
```

Reply:
> I feel like it should be like

Thanks so much! It looks better and prettyer